canonical-web-and-design/maas-ui
# Build third party drivers form. Fixes #1402

## Done
Build form for "enable third party drivers"

## QA
- Go to `/settings/images/ubuntu`
- Change the value of the checkbox and save
- Reload and see that the change persists
- Go to old MAAS settings (`:5240/MAAS/settings/general/`)
- See that the checkbox under **Third Party Drivers Configuration** reflects the same value
